allennlp/training/trainer.py

```diff
@@ -32,7 +32,8 @@ def __init__(self,
                  cuda_device: int = -1,
                  grad_norm: Optional[float] = None,
                  grad_clipping: Optional[float] = None,
-                 log_one_line_per_batch: bool = False) -> None:
+                 log_one_line_per_batch: bool = False,
+                 validation_metric_decreases: bool = True) -> None:
```

Comment:
I don't think that's easier or clearer - it would mean that the default `loss` would have to have that feature and mean that you can have validation_metric keys which need to be matched to keys in the metric dictionary. This is more obvious. @joelgrus break the tie?